app/javascript/components/Pages/IcebreakerAnswer.js

```diff
@@ -96,15 +97,17 @@ const IcebreakerAnswer = ({data, setData, saveDataToDb, steps, service}) => {
                 </div>
               </div>
             </div>
-            <div className='d-flex justify-content-between m-3'>
-              <ShoutOutIcon/>
-              <BtnBack addClass="answer-custom" onClick={backHandling}/>
-              <BtnPrimary addClass="answer-custom" onClick={handlingOnClickNext} text={isEmptyStr(answerBody) ? 'Skip to Results' : 'Submit'} />
-              <HelpIcon/>
+            <div className='d-flex placement-buttons justify-content-between col-6 offset-3 pb-53 mt-5'>
```

Comment:
the same - **col-6 offset-3**. maybe w-50 is better here?

Reply:
I think this decision more preferable in this case.